### ISSUE TITLE
Suggestion - Backup option

### ISSUE THREAD

**Issue by mdrodge** [kylepaulsen/SuperBookmarkDesktop#16]
Can we get an option to save the config/layout/everything to a file/folder/whatever
I'm about to format my drive but before I do I have no other way to backup than to save every link manually.

I store all my backups on my server but this is plugin doesn't seem to have that option.

If they are synced somewhere then ok bud I'd have no way to know that until it's to late. (also I use it on brave and I don't know if that would work)
Basically I'd rather have the piece of mind that come from a physical backup (cloud is just someone else's server)

Apart from that i love this extension

**Comment by kylepaulsen** [comment]
I know it's insanely late for this, but this is now an option in v1.5.0. Should be on the Chrome Web Store soon.
